tests/io_components/test_utils.py

```diff
@@ -71,6 +71,7 @@ def testcombine_metadata_lists():
         (("a", "abc", None), ("a", ["abc"], [])),
         (("a", None, "abc"), ("a", [], ["abc"])),
         (("a", None, None), ("a", [], [])),
+        (("a", ("abc"), {"abc"}), ("a", ["a"], ["a"])),
```

Comment:
Expected should probably be `("a", ["abc"], ["abc"])`, shouldn't it?

Reply:
Yes sorry, but I re-read the bug description and now raise an expection if sets or tuples are given as output. Am I going in the right direction?